pkg/monitoring/metrics/virt-controller/vmstats_collector_test.go

```diff
@@ -30,11 +31,83 @@ import (
 	k8sv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
 
 	k6tv1 "kubevirt.io/api/core/v1"
 )
 
 var _ = Describe("VM Stats Collector", func() {
+	Context("PVC allocated size metric collection", func() {
+		BeforeEach(func() {
+			persistentVolumeClaimInformer = cache.NewSharedIndexInformer(
```

Comment:
unit tests are failing because you are probably incorrectly configuring the informer for the collector, check the implementation in https://github.com/kubevirt/kubevirt/blob/d7420c6e2eb3a3029847b36ffaa48bb0cdf8b00e/pkg/monitoring/metrics/virt-controller/vmistats_collector_test.go#L268